### DISCUSSION TITLE
Click and navigate to components

### DISCUSSION THREAD

**Discussion by damevin** [nuxtrdev/nuxtr-vscode#2] [Ideas]
Hey everyone, thanks for the work !!

I wanted to share a really cool new feature that I think will be super useful for working with Vue projects in VSCode. Now, you can navigate directly to components by clicking on them!

If you've ever used Vue, you might be familiar with this feature already, but with Nuxt it hasn't always been possible. For example, if you had a component called `CommonButtonPrimary` that was auto-imported, you would need to go to the `Primary.vue` file in the Button folder of the Common directory to see the component code.

But now, with this new feature, you can simply cmd + click on the component name in your code editor and it will take you directly to the file where it's located. This makes it so much easier to navigate your project and find the components you need, especially if you're working with a large codebase.

I've already tried this out and it's been a game-changer for me. I hope you find it just as useful! Let me know if you have any questions or if you've tried it out yourself.

**Comment by adhamfarrag** [comment]
Hey @damevin! First idea from our users! This made our day :) 

Really cool idea! Will add it to our roadmap and please keep your ideas coming!
